autoload/leaderf/Filer.vim

```diff
@@ -85,43 +74,25 @@ function! leaderf#Filer#InsertMap() abort
     let l:default_map = {}
     if get(g:, 'Lf_FilerUseDefaultInsertMap', v:true)
         let l:default_map = {
-        \   '<C-h>':        'open_parent_or_clear_line',
```

Comment:
As with normal mode, insert mode can be customized by using `g:Lf_FilerInsertMap`, so we don't need to change the default mapping.